apps/fhir/bluebutton/utils.py

```diff
@@ -1,5 +1,7 @@
 import os
-import logging
+
+import apps.logging.request_logger as logging
```

Comment:
Can't leave the comment on the right line but we have access to the `request` object in these functions. So we'll need to move down the logger get so that it can include the `request`. 
Line 244: logger_perf.info(header_detail)